core/chains/evm/txmgr/eth_broadcaster.go

```diff
@@ -89,7 +89,7 @@ type TransmitChecker[ADDR types.Hashable[ADDR], TX_HASH types.Hashable[TX_HASH]]
 // - a monotonic series of increasing nonces for eth_txes that can all eventually be confirmed if you retry enough times
 // - transition of eth_txes out of unstarted into either fatal_error or unconfirmed
 // - existence of a saved eth_tx_attempt
-type EthBroadcaster[ADDR types.Hashable[ADDR], TX_HASH types.Hashable[TX_HASH], BLOCK_HASH types.Hashable[BLOCK_HASH]] struct {
+type EthBroadcaster[ADDR types.Hashable, TX_HASH types.Hashable, BLOCK_HASH types.Hashable] struct {
 	logger    logger.Logger
 	txStore   txmgrtypes.TxStore[ADDR, big.Int, TX_HASH, BLOCK_HASH, NewTx[ADDR], *evmtypes.Receipt, EthTx[ADDR, TX_HASH], EthTxAttempt[ADDR, TX_HASH], int64, int64]
 	ethClient evmclient.Client
```

Comment:
There's another field, which uses string for map types here:
`triggers map[string]chan struct{}`
Can you change it to ADDR too?

Reply:
will do!

Reply:
so this change propagated a little bit - the broadcaster now has a generic func for parsing string payloads into the address type which is currently evm specific but will be cleaned up in the future